app/upgrades.go

```diff
@@ -346,6 +349,33 @@ func updateMaxSupply(ctx sdk.Context, app *App) {
 	ctx.Logger().Info("Done updating MaxSupply marker param")
 }
 
+// addMarkerNavs adds navs to existing markers with default value of 15 cents
+// TODO: Need to get a list of current markers and their values to set as a manual script
+func addMarkerNavs(ctx sdk.Context, app *App) {
+	ctx.Logger().Info("Adding marker net asset values")
+	//TODO: Add list of actual net asset values to set here
+
+	app.MarkerKeeper.IterateMarkers(ctx, func(record markertypes.MarkerAccountI) bool {
+		var hasNav bool
+		err := app.MarkerKeeper.IterateNetAssetValues(ctx, record.GetAddress(), func(nav markertypes.NetAssetValue) bool {
+			hasNav = true
+			return true
+		})
+		if err != nil {
+			ctx.Logger().Error(fmt.Sprintf("unable iterate net asset values for marker %v: %v", record, err))
+			return false
+		}
+		if !hasNav {
```

Comment:
I don't think this is a good default ... we could set a default for HASH on testnet to a NAV of 15¢ (1:1500000000) as a good example of what we expect a token with a 1 billion base unit to have against a cent denom.

Reply:
We should delete this `!hasNav` section unless we have a default we use for new markers which we could set here to match...